upb/decode_internal.h

```diff
@@ -142,19 +142,22 @@ bool decode_isdone(upb_decstate *d, const char **ptr) {
   }
 }
 
+#if UPB_FASTTABLE
 UPB_INLINE
 const char *fastdecode_tagdispatch(upb_decstate *d, const char *ptr,
                                     upb_msg *msg, intptr_t table,
-                                    uint64_t hasbits, uint32_t tag) {
+                                    uint64_t hasbits, uint64_t tag) {
```

Comment:
should this be the params macro?

Reply:
Currently the params macro is not available in the header. I started shuffling some things around so that it would be, but it cascaded a bit and I think I'd prefer to do it in a separate change.

Reply:
SGTM